src/icon-manifest/supportedExtensions.ts

```diff
@@ -573,6 +574,14 @@ export const extensions: IFileCollection = {
     },
     { icon: 'vsix', extensions: ['vsix'], light: true, format: FileFormat.svg },
     { icon: 'vue', extensions: [], languages: [languages.vue], format: FileFormat.svg },
+    {
+      icon: 'wallaby',
+      extensions: [],
+      filenamesGlob: ['wallaby', 'wallaby.conf', '.wallaby', '.wallaby.conf'],
+      extensionsGlob: ['json', 'js', 'ts', 'coffee'],
+      filename: true,
+      format: FileFormat.svg,
+    },
```

Comment:
?

Does this branch bases on the wallaby branch?

Reply:
Damn, I thought I corrected that. Sorry. Good catch.